fineract-provider/src/main/java/org/apache/fineract/infrastructure/gcm/domain/Sender.java

```diff
@@ -789,7 +745,7 @@ protected static String getString(InputStream stream) throws IOException {
 			return "";
 		}
 		BufferedReader reader = new BufferedReader(
-				new InputStreamReader(stream));
+				new InputStreamReader(stream, Charset.defaultCharset()));
```

Comment:
hard-code this to use UTF-8 instead of using platform default charset is better here, as it make it more repeatable (otherwise it depends on OS settings, which can lead to hard to understand problems)